server/server.go

```diff
@@ -231,3 +290,34 @@ func (server *Server) getStylesHandler(c *gin.Context) {
 
 	c.JSON(http.StatusOK, gin.H{"styles": common.GsStyles})
 }
+
+func (server *Server) findBoard(board string) bool {
```

Comment:
findBoard is not correct for me because the board is not returned. Method verify only if board exist.